lib/docusign_rest/client.rb

```diff
@@ -1674,7 +1674,7 @@ def get_envelope_id_options(input)
 
       {
         add_envelope_tab: true,
-        envelope_id_tabs: [{ anchorString: input[:anchor_string], bold: false }],
+        envelope_id_tabs: [{ anchorString: input[:anchor_string], bold: false, fontSize: 'size8' }],
```

Comment:
is the size case sensitive? In documentation it has `Size8` instead of `size8`

Reply:
Not case sensitive, verified it works.